__tests__/conformers.js

```diff
@@ -75,4 +75,21 @@ describe('ConformerGenerator', () => {
     expect(conf).toBe(mol);
     expect(mol.getAtomZ(0)).not.toBe(0);
   });
+
+  it('should be an iterator', () => {
+    const mol = Molecule.fromSmiles('COCCON');
+
+    const gen = new ConformerGenerator(1);
+    gen.initializeConformers(mol);
+
+    const allConformers = [...gen];
```

Comment:
of course `for...of` can also be used, and `break` at any time. This is for the test.